src/homemate_bridge/cli.py

```diff
@@ -355,31 +363,30 @@ def handle_state_update(self, packet):
         return None  # No response to this packet
 
     def handle_handshake(self, packet):
-        if 'localIp' in packet.json_payload and packet.json_payload['localIp'] in self.__class__._device_settings:
-            # By default, we try to use the source IP of the socket as a stable identifier when connecting to HA
-            # This may not always be the right thing to do (ie, if there is NAT involved, like when running in docker)
-            # Fortunately, the switch sends the localIP in cmd 6, which happens before the cmd 32 wait for before
-            # Connecting to MQTT
-
-            localip = packet.json_payload['localIp']
-            self.entity_id = localip.replace('.', '_')
-            logger.debug("Updating device settings for {}, localIp={}".format(self.switch_id, localip))
-            self.settings = self.__class__._device_settings[localip]
-            if 'name' not in self.settings:
-                self.settings['name'] = "Homemate Switch " + localip
-
-            logger.debug("Device settings: {}".format(self.settings))
```

Comment:
Why remove? This is important for my use-case; as I run my bridge in a Kubernetes cluster, so the connections to the bridge are NATed, and appear to all come from the same IP. I use the IP that the device itself reports as an ID so I can tell them apart

Reply:
Ah.